src/platform/intel/cavs/platform.c

```diff
@@ -440,6 +440,16 @@ int platform_init(struct sof *sof)
 
 	/* initialize PM for boot */
 
+#if CAVS_VERSION == CAVS_VERSION_2_5
+	/* Check minimal HW version supported */
+
+	shim_write(SHIM_CLKCTL, shim_read(SHIM_CLKCTL) | SHIM_CLKCTL_WOV_CRO_REQUEST);
+	if (shim_read(SHIM_CLKCTL) & SHIM_CLKCTL_WOV_CRO_REQUEST)
+		shim_write(SHIM_CLKCTL, shim_read(SHIM_CLKCTL) & ~SHIM_CLKCTL_WOV_CRO_REQUEST);
+	else
+		panic(SOF_IPC_PANIC_PLATFORM);
```

Comment:
do wen really need panic here? can we just not use WOVCRO if it is not available?

Reply:
Yes, we can, that was just our early plan to stop using not compatible steppings. But we can still discuss this, what do you think @mmaka1  about @keyonjie idea?

Reply:
We should actually report an error here (lower than panic but higher than normal). Ideally we should tell the driver with a new IPC so driver code can reset DSP HW etc and try and recover. i.e. any error that FW cannot recover we should send up stack so driver can attempt recovery.